automation/taskcluster/lib/tasks.py

```diff
@@ -130,7 +130,7 @@ def craft_test_task(self, variant):
     def _craft_branch_routes(self, variant):
         routes = []
 
-        if self.repo_url == _OFFICIAL_REPO_URL and self.branch == 'master':
+        if self.repo_url == _OFFICIAL_REPO_URL and self.branch == 'refs/heads/master':
```

Comment:
Why is `self.branch` prefixed with `refs/heads`?
Perhaps it isn't a "branch" in that case? Perhaps it should be named `self.ref` instead?

Reply:
Yeah, `self.ref` would make better sense. We do use refs https://github.com/mozilla-mobile/fenix/pull/1317/files#diff-ac0229d1171c0983b27003af4c7441a5L20

Reply:
Done. I used `self.git_ref` because I found `self.ref` too ambiguous.